collect_app/src/main/res/values-v21/themes.xml

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-
-<!--
-    This file contains the app's light and dark theme definitions for
-    API 21 and above
```

Comment:
Why did you remove this file? The message is clear and i explained above:

> if you have resources with specified API level like here v17 it's not only for API 17 but for 17+. I know it's misleading but actually this file is the one we want to keep, it should be moved to general layout folder to replace the one that exists there and then this folder should be removed.

but in all cases you removed resources we need.